packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -210,7 +210,7 @@ export async function load_node({
 
 							// TODO arrayBuffer?
 
-							return Reflect.get(response, key, receiver);
```

Comment:
This is not Workers-specific. It's an illegal invocation in a browser runtime, too. For example:

```js
let res = new Response('hello', { status: 201 });
let proxy1 = new Proxy(res, {
    get(r, k, rec) {
        console.log({ r, k, rec });
        return Reflect.get(r, k, rec);
    }
});

proxy1.status;
// logs: {r: Response, k: "status", rec: Proxy}
// throws: Uncaught TypeError: Illegal invocation

let proxy2 = new Proxy(res, {
    get(r, k) {
        console.log({ r, k });
        return Reflect.get(r, k, r);
        // or return Reflect.get(r, k);    
    }
});

proxy2.status;
// logs: {r: Response, k: "status" }
//=> 201
```

Reply:
Oh, interesting, thanks. I guess `node-fetch` is the thing with aberrant behaviour. The `Headers` issue stands though AFAICT

Reply:
Yup, I forwarded the report. Thanks again for raising it 👍